pkg/controller/pxc/controller.go

```diff
@@ -153,6 +153,17 @@ type CronRegistry struct {
 	backupJobs        *sync.Map
 }
 
+// AddFuncWithSeconds does the same as cron.AddFunc but changes the schedule so that the function will run the exact second that this method is called.
+func (r *CronRegistry) AddFuncWithSeconds(spec string, cmd func()) (cron.EntryID, error) {
+	schedule, err := cron.ParseStandard(spec)
+	if err != nil {
+		return 0, errors.Wrap(err, "failed to parse cron schedule")
+	}
+	schedule.(*cron.SpecSchedule).Second = uint64(1 << time.Now().Second())
```

Comment:
I don't understand why we're doing this

Reply:
[SpecSchedule](https://pkg.go.dev/github.com/robfig/cron#SpecSchedule) stores its values as bit sets. 
We use a standard parser, that doesn't accept seconds in the schedule string and always sets `SpecSchedule.Seconds` to `uint64(1 << 0)`. That means that every job will be executed at 0 second.

On this line, we set the current time seconds. Please check [new PR description](https://github.com/percona/percona-xtradb-cluster-operator/pull/1233#issue-1408890321)